### PR TITLE
[Travis] Switched to using Flex for testing

### PR DESCRIPTION
**Pull request by mnocon**
JIRA: https://issues.ibexa.co/browse/EZP-32201

Switched to Flex installation for test setup.

I had to change the `bin` directory in Composer to make thing unified.

Also I had troubles getting the Scenario with customised use group to work: https://github.com/ezsystems/ezplatform-content-forms/blob/master/features/User/Registration/user_registration.feature#L20-L34

For some reason it doesn't work when other config than the one from metarepository was used, I've decided to add it here and copy it to correct location before tests are started - I didn't want to disable this test, this keeps this test running while we can investigate it in the future.

### CODE REVIEW

File: bin/.travis/prepare_unittest.sh [hunk 1]
```diff
@@ -1,19 +0,0 @@
-#!/bin/sh
```
Comment (mnocon) [inline bin/.travis/prepare_unittest.sh:1]: This script has been removed as it was doing anything relevant anymore

File: .travis.yml [hunk 2]
```diff
@@ -55,13 +55,13 @@ before_install:
 
 # setup requirements for running unit/behat tests
 install:
-  - if [ "$CHECK_CS" != "" ] ; then travis_retry composer install --prefer-dist --no-interaction --no-suggest ; fi
-  - if [ "$TEST_CONFIG" != "" ] ; then ./bin/.travis/prepare_unittest.sh ; fi
-  - if [ "$BEHAT_OPTS" != "" ]; then ./bin/.travis/prepare_ezplatform.sh ; fi
+  - travis_retry composer install --prefer-dist --no-interaction
+  - if [ "${BEHAT_OPTS}" != "" ]; then ./vendor/bin/prepare_project_edition.sh oss ${PROJECT_VERSION} ${COMPOSE_FILE} ; fi
+  - if [ "$BEHAT_OPTS" != "" ] ; then cd "$HOME/build/project"; docker-compose exec --user www-data app sh -c "mkdir -p config/packages/behat/" ; fi
```
Comment (mnocon) [inline .travis.yml:60]: Right now this directory does not exist - I create it manually for these tests to pass, we might add a recipe that creates it in the future (but it's a bit too early IMHO to do that, and an artificial recipe just to create a directory seems like an overkill).

File: .travis.yml [hunk 3]
```diff
@@ -55,13 +55,14 @@ before_install:
 
 # setup requirements for running unit/behat tests
 install:
-  - if [ "$CHECK_CS" != "" ] ; then travis_retry composer install --prefer-dist --no-interaction --no-suggest ; fi
-  - if [ "$TEST_CONFIG" != "" ] ; then ./bin/.travis/prepare_unittest.sh ; fi
-  - if [ "$BEHAT_OPTS" != "" ]; then ./bin/.travis/prepare_ezplatform.sh ; fi
+  - travis_retry composer install --prefer-dist --no-interaction
+  - if [ "${BEHAT_OPTS}" != "" ]; then ./vendor/bin/prepare_project_edition.sh oss ${PROJECT_VERSION} ${COMPOSE_FILE} ; fi
```
Comment (bogusez) [inline .travis.yml:59]: space needed

**Review by bogusez — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 3 above)
Comment (mnocon) [reply .travis.yml]: Thanks, fixed!

**Review by bogusez — APPROVED** [review]

**Pull request merged**